indico/modules/events/papers/templates/management/assignment.html

```diff
@@ -0,0 +1,95 @@
+{% extends 'layout/full_content_base.html' %}
+{% from 'events/papers/management/_paper_assignment_list.html' import render_paper_assignment_list %}
+{% from 'events/management/_lists.html' import render_displayed_entries_fragment %}
+
+{% block title %}
+    {% trans %}Paper Assignment{% endtrans %}
+{% endblock %}
+
+{% block description %}
+    {%- trans -%}
+        Assign <strong>judges</strong>, <strong>content reviewers</strong> and <strong>layout reviewers</strong>
+        to contributions in order to allow assessing its paper revisions.
+    {%- endtrans -%}
+{% endblock %}
+
+{% block content %}
+    <div class="toolbars space-after">
+        <div class="toolbar">
+            <div class="group">
+                <a class="i-button icon-checkbox-checked arrow js-dropdown" data-toggle="dropdown"></a>
+                <ul class="dropdown">
+                    <li>
+                        <a href="#" data-select-all="#assignment-list input:checkbox">{% trans %}All{% endtrans %}</a>
+                    </li>
+                    <li>
+                        <a href="#" data-select-none="#assignment-list input:checkbox">{% trans %}None{% endtrans %}</a>
+                    </li>
+                </ul>
+            </div>
+            <div class="group">
+                <button class="i-button icon-settings js-dialog-action js-customize-list highlight"
+                        data-href="#"
+                        data-title="{% trans %}Paper assignment list configuration{% endtrans %}"
+                        data-dialog-classes="list-filter-dialog"
+                        data-update='{"html": "#assignment-list", "filter_statistics": "#filter-statistics"}'
+                        data-ajax-dialog>
+                    {% trans %}Customize list{% endtrans %}
+                </button>
+            </div>
+            <div class="group">
+                <a class="i-button arrow js-dropdown" data-toggle="dropdown">
+                    {%- trans %}Assign{% endtrans -%}
+                </a>
+                <ul class="dropdown">
+                    <li><a href="#">{% trans %}Judge{% endtrans %}</a></li>
+                    <li><a href="#">{% trans %}Content reviewer{% endtrans %}</a></li>
+                    <li><a href="#">{% trans %}Layout reviewer{% endtrans %}</a></li>
+                </ul>
+                <a class="i-button">
+                    {%- trans %}Unassign{% endtrans -%}
+                </a>
+            </div>
+            <div class="group">
+                <a href="#" class="i-button icon-checkmark js-enable-if-checked disabled"
+                   data-href="#">
+                    {%- trans %}Accept{% endtrans -%}
```

Comment:
Just wondering if sending papers back for corrections make sense... Shouldn't we enforce leaving some feedback for the submitter?